src/app/Dashboard/AutomatedAnalysis/ClickableAutomatedAnalysisLabel.tsx

```diff
@@ -92,6 +92,20 @@ export const ClickableAutomatedAnalysisLabel: React.FunctionComponent<ClickableA
       : 'danger';
   }, [label.score]);
 
+  const icon = React.useMemo(() => {
+    return label.score == AutomatedAnalysisScore.NA_SCORE ? (
+      <InfoCircleIcon />
+    ) : label.score < AutomatedAnalysisScore.ORANGE_SCORE_THRESHOLD ? (
+      <CheckCircleIcon />
+    ) : label.score < AutomatedAnalysisScore.RED_SCORE_THRESHOLD ? (
+      <WarningTriangleIcon />
+    ) : (
+      <ExclamationCircleIcon />
+    );
+  }, [label.score]);
+
+  const [summary, explanation] = label.description.split('Explanation:');
```

Comment:
Could u wrap this part in a function? In case JMC upgrades, we can easily trace it to change? 

Also, we need to replace `\n` with empty spaces and trim trailing/leading spaces too. Response description (raw string) includes these newline chars.

Reply:
The `Summary:` and `Explanation:` parts in the text are actually being placed there in `cryostat-core`'s report generation. JMC supplies those as separate properties, but our data path here only supports a single text property to display here, so I joined them together like this.

Reply:
https://github.com/cryostatio/cryostat-core/commit/2336321faa240d3c4e6da5d5ec22062d74673a2b#diff-9f4e83d307a0c834eb25732bd9c56dc920c2a4869c5611143864ee0d85d384faR227

Reply:
Splitting on the `Explanation:` may be incorrect though, since there are Summary, Explanation, Solution, and Suggested Settings components, all of which are optional. If the double-newline characters are still present in the JSON response, which I think they probably are, then it might be better to just split on those, and for each split element, highlight/bold from the beginning until the first `:`.

Reply:
is localization supported in these JMC funtionalities? Or we need to build our own translation?

Reply:
JMC does localize this and the rest of the strings you can see in our view here are in fact coming from JMC localization files for the different rules. I'm not sure why it's broken for some ex. that `SocketREadRuleFactory_TEXT_NO_EVENTS`, but the rest of the text around it is properly being localized.

We'd need to figure out how to set the locale on the JMC code we're calling into, and then set up some way to get the web-client user's locale setting included with the API request (a header probably), passed to the Cryostat server, and then either handled by the `SubprocessReportGenerator` or passed along again to a `cryostat-reports` instance (header again). In either of those cases the header value needs to be used to set that JMC locale so that the generated strings, hopefully, match the web-client user's locale.

```bash
~/workspace/jmc/core $ ag SocketReadRuleFactory_TEXT_NO_EVENTS
org.openjdk.jmc.flightrecorder.rules.jdk/src/main/java/org/openjdk/jmc/flightrecorder/rules/jdk/io/SocketReadRule.java
135:					.setSummary(Messages.SocketReadRuleFactory_TEXT_NO_EVENTS)

org.openjdk.jmc.flightrecorder.rules.jdk/src/main/java/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/Messages.java
539:	public static final String SocketReadRuleFactory_TEXT_NO_EVENTS = "SocketReadRuleFactory_TEXT_NO_EVENTS"; //$NON-NLS-1$

org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages_zh_CN.properties
530:SocketReadRuleFactory_TEXT_NO_EVENTS=\u6B64\u8BB0\u5F55\u4E2D\u6CA1\u6709\u4EFB\u4F55\u5957\u63A5\u5B57\u8BFB\u53D6\u4E8B\u4EF6\u3002

org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages_ja.properties
615:SocketReadRuleFactory_TEXT_NO_EVENTS=\u3053\u306E\u8A18\u9332\u306B\u306F\u30BD\u30B1\u30C3\u30C8\u8AAD\u53D6\u308A\u30A4\u30D9\u30F3\u30C8\u306F\u3042\u308A\u307E\u305B\u3093\u3002

org.openjdk.jmc.flightrecorder.rules.jdk/src/main/resources/org/openjdk/jmc/flightrecorder/rules/jdk/messages/internal/messages.properties
617:SocketReadRuleFactory_TEXT_NO_EVENTS=There are no socket read events in this recording.

tests/org.openjdk.jmc.flightrecorder.rules.jdk.test/src/main/resources/baseline/JfrRuleBaseline.xml
2125:            <summary>SocketReadRuleFactory_TEXT_NO_EVENTS</summary>
2415:            <summary>SocketReadRuleFactory_TEXT_NO_EVENTS</summary>
3971:            <summary>SocketReadRuleFactory_TEXT_NO_EVENTS</summary>

~/workspace/jmc/core $ jshell
|  Welcome to JShell -- Version 17.0.6
|  For an introduction type: /help intro

jshell> var s = "\u3053\u306E\u8A18\u9332\u306B\u306F\u30BD\u30B1\u30C3\u30C8\u8AAD\u53D6\u308A\u30A4\u
30D9\u30F3\u30C8\u306F\u3042\u308A\u307E\u305B\u3093\u3002"
s ==> "この記録にはソケット読取りイベントはありません。"

jshell> var t = "\u6B64\u8BB0\u5F55\u4E2D\u6CA1\u6709\u4EFB\u4F55\u5957\u63A5\u5B57\u8BFB\u53D6\u4E8B\u
4EF6\u3002"
t ==> "此记录中没有任何套接字读取事件。"

jshell> 
```

It looks like JMC currently has localizations for English, Chinese, and Japanese. If the web-client user requests something else then we need to fall back to English, as usual, though this might be a bit annoying for the user if there are mismatches between our set of localizations and JMC's.

Reply:
It looks like JMC is handling this with good old Java `MessageFormat`, `ResourceBundle`, etc. From what I can tell it just lets the `ResourceBundle` use the default locale that the JVM picked on startup, which it would have determined from environment variables etc. We will probably need to do some hacking, or offer some patches to JMC for their Rules package, so that it's possible to get these rules report results with an override for the locale, in case the JVM is serving results to remote users like in our case. Otherwise if we need to hack it, then I'm not sure the threadsafety of setting the JVM's global locale, but maybe we can explore options around there.

Reply:
Anyway @maxcao13 as far as this PR is directly concerned, I think just splitting on the hardcoded `Summary:`, `Explanation:`, etc. is good enough. Just need to make sure each of those being potentially missing from the API response is handled.

Reply:
Okay, makes sense to me, refitting this for localizations can come next patch.

Reply:
@tthvo or @maxcao13 file an issue for the above discussion, please. I think this should be a Story or even an Epic on the main repo, and then there will be bits and pieces to complete for the main repo, -core, -web, and -reports to get this all done.

Reply:
https://github.com/cryostatio/cryostat/issues/1464